events-api/src/shared/middleware/logger.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+  use(req: Request, _res: Response, next: NextFunction) {
+    console.log(`[${new Date().toISOString()}] ${req.method}: ${req.baseUrl + req.url}`);
```

Comment:
You can use build-in NestJS logger instead of console log for example

Reply:
fixed